code/modules/mob/living/carbon/human/species_types/ethari.dm

```diff
@@ -0,0 +1,38 @@
+/datum/species/ethari
+	// Furry people with bushy tails and stuff.
+	name = "Ethari"
+	id = "ethari"
+	default_color = "#DF8134"
+	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,HAIR)
+	mutant_bodyparts = list("tail_ethari", "body_markings", "snout_ethari", "ears_ethari")
+	coldmod = 0.67
+	heatmod = 1.5
+	brutemod = 1.2
+	burnmod = 1.2
```

Comment:
Do we definitely want 20% extra burn and brute for these guys? What's the reasoning behind it?

Reply:
fukken furries reeeee

is the reasoning

Reply:
^ more or less. Won't have people complaining about snowflakes if they're clearly inferior. I don't think it's that impactful, considering 95% of the game doesn't involve bashing someone's skull in, but when it comes to it, it's a weakness you have to consider.